control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -631,6 +633,34 @@ func (r *reconciler) reconcileConfig(ctx context.Context, hcp *hyperv1.HostedCon
 	return errors.NewAggregate(errs)
 }
 
+func (r *reconciler) reconcileProxyTrustedCAConfigMap(ctx context.Context, hcp *hyperv1.HostedControlPlane) error {
+	if hcp.Spec.Configuration == nil || hcp.Spec.Configuration.Proxy == nil || len(hcp.Spec.Configuration.Proxy.TrustedCA.Name) == 0 {
```

Comment:
This is not specific to this particular reconciliation loop, but for the record what happens if the the .Configuration.Proxy.TrustedCA.Name is deleted?

Reply:
the proxy resource will be updated only, and this loop will be skipped. Should we cleanup previously referenced configmaps?